package.json

```diff
@@ -4,15 +4,15 @@
     "packages/*"
   ],
   "scripts": {
-    "build": "echo Use synthetic-chain to build proposal images",
-    "test": "echo Use synthetic-chain to test proposal images",
+    "build": "./node_modules/.bin/synthetic-chain build",
+    "test": "./node_modules/.bin/synthetic-chain test",
     "format": "prettier --write ."
   },
   "dependencies": {
     "@agoric/synthetic-chain": "workspace:*"
   },
   "agoricSyntheticChain": {
-    "fromTag": null
+    "fromTag": "main"
```

Comment:
this is a vestige of my local testing. it only works if you also delete all the _passed_  proposals (because they're already in `main`) 

Before merge I'll have to set this back to `null`.